app/workers/scheduler/personalized_for_you_statuses_scheduler.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require 'http'
+require 'json'
+
+class Scheduler::PersonalizedForYouStatusesScheduler
+  FOR_YOU_OWNER_ACCOUNT = ENV['FOR_YOU_OWNER_ACCOUNT'] || 'admin'
+  BETA_FOR_YOU_LIST = 'Beta ForYou Personalized'
+
+  # Get Statuses for the last n hours for a Personalized 'For You' Feed
+  # Iterate over and task a worker to fetch the status from original source
+  # Update the Status Stat for boosted & likes for that status
+  include Sidekiq::Worker
+
+  sidekiq_options retry: 0
+
+  def perform
+    # Get accounts on beta list
+    # For each account get fedigraph of follows
+    # For each follow account get statuses and send to for_you_worker
+    for_you_personalized_beta_list_accounts.each do |account|
+      statuses_for_indirect_follows(account).each do |status|
+        ForYouFeedWorker.perform_async(status['id'], account.id, 'personal')
+      end
+    end
+  end
+
+  private
+
+  def for_you_personalized_beta_list_accounts
+    default_owner_account = Account.local.where(username: FOR_YOU_OWNER_ACCOUNT).first!
+    beta_list = List.where(account: default_owner_account, title: BETA_FOR_YOU_LIST).first!
+    beta_list.accounts.without_suspended.includes(:account_stat)
+  end
+
+  # Indirect Follows are the following of your followers
+  # IE Friends of Friends
+  def statuses_for_indirect_follows(account)
```

Comment:
Good name choice here.

Reply:
Stole that from Felix in the Follow Recommendations 😉